Gemfile

```diff
@@ -1,18 +1,18 @@
 source "https://rubygems.org"
 
 group :test do
-  gem "rake"
-  gem "puppet", ENV['PUPPET_VERSION'] || '~> 3.7.0'
+  gem "rake", '~> 11.2.2'
+  gem "puppet", ENV['PUPPET_VERSION'] || '~> 4.5.3'
```

Comment:
if puppet is locked at 3.3.0 through the travis file, shouldn't it be that here as well?